server/middlewares/signupValidations/passwordValidation.js

```diff
@@ -0,0 +1,36 @@
+import helpers from './validationHelper';
+/**
+ * @description This class is for validating email and password fields
+ */
+class PasswordValidation {
+  /**
+   * @description A middleware for validating email
+   * @param {object} req - get response from body
+   * @param {object} res - response to be sent
+   * @param {object} next - callback function
+   * @returns {object} A response object
+   */
+  static isPasswordCheck(req, res, next) {
```

Comment:
This function name can be better. Your function name should clearly state what it does